docs/reference.md

```diff
@@ -1174,7 +1174,32 @@ Implicit labels that can be used/exposed:
   - COPYBARA_AUTHOR: The author of the change
 
 
-`core.workflow(name, origin, destination, authoring, transformations=[], origin_files=glob(["**"]), destination_files=glob(["**"]), mode="SQUASH", reversible_check=True for 'CHANGE_REQUEST' mode. False otherwise, check_last_rev_state=True for CHANGE_REQUEST, ask_for_confirmation=False, dry_run=False, after_migration=[], after_workflow=[], change_identity=None, set_rev_id=True, smart_prune=False, migrate_noop_changes=False, experimental_custom_rev_id=None, description=None, checkout=True, reversible_check_ignore_files=None)`
+```python
```

Comment:
Thank you for sending the contribution. Note that the reference is auto-generated by the java/com/google/copybara:generate_reference rule.

Cheers,
~H

Reply:
Then there is no hope for a merge. OK. I'll just close.